lib/text_config.dart

```diff
@@ -88,9 +88,12 @@ class TextConfig {
     Vector2 p, {
     Anchor anchor = Anchor.topLeft,
   }) {
-    final material.TextPainter tp = toTextPainter(text);
-    final Vector2 translatedPosition = anchor.translate(p, tp.size.toVector2());
-    tp.paint(canvas, translatedPosition.toOffset());
+    final material.TextPainter? tp = toTextPainter(text);
```

Comment:
doesn't toTextPainter returns non-null?

Reply:
You're right. I looked at the check after that and put that there. Removing nullable and the check.